packages/client/tests/functional/composites-object/count.ts

```diff
@@ -0,0 +1,52 @@
+import { faker } from '@faker-js/faker'
+import pRetry from 'p-retry'
+
+import { setupTestSuite } from './_matrix'
+
+// @ts-ignore this is just for type checks
+declare let prisma: import('@prisma/client').PrismaClient
+
+const id = faker.database.mongodbObjectId()
+
+setupTestSuite(() => {
+  beforeAll(async () => {
+    await pRetry(
```

Comment:
WDYT about removing the retry logic? Do you know why we are doing it?

Reply:
I believe we should be able to remove it while we are running tests in band. As soon as `workers: 1` is removed, we will need to make sure that there is proper database isolation, otherwise the problem will come back.